src/Stryker.CLI/Stryker.CLI/OptionsBuilder.cs

```diff
@@ -37,6 +37,7 @@ public OptionsBuilder(ILogger logger)
             CommandOption coverageAnalysis,
             CommandOption abortTestOnFail,
             CommandOption configFilePath,
+            CommandOption runMultipleMutations,
```

Comment:
```suggestion
            CommandOption disableSimultaneousTesting,
```

Reply:
thanks for the fixes. it seems I forgot some along the way, indeed